.github/workflows/check-try-state.yml

```diff
@@ -24,17 +24,47 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
 
       - name: Setup node
         uses: actions/setup-node@v4
         with:
           node-version: 20.x
 
       - name: Install system dependencies
-        run: sudo apt -y install protobuf-compiler
+        run: |
+          sudo apt -y install protobuf-compiler jq
+          curl -L https://github.com/vi/websocat/releases/latest/download/websocat.x86_64-unknown-linux-musl -o websocat
```

Comment:
You mentioned it doesn't work for the secret WSS endpoint, but do you need to use _that_ one?

In case of e.g. try-runtime tests we want to use something without limits.
But for a single RPC request, public endpoint should be just fine, right?

Reply:
You're right, I hadn't considered this approach, but it makes much more sense